e2e/cypress/integration/enterprise/system_console/compliance/download_bot_compliance_export_file_spec.js

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @system_console
+
+const path = require('path');
+
+const ExportFormatActiance = 'Actiance XML';
+let targetDownload;
+let fileURL;
+let pwd;
+
+describe('Compliance Export', () => {
+    let newTeam;
+    let newChannel;
+    let botId;
+    let botName;
+
+    before(() => {
+        cy.exec('PWD').then((result) => {
+            pwd = result.stdout;
```

Comment:
`pwd` is not used elsewhere and so can declare here.
```suggestion
            const pwd = result.stdout;
```